content/docs/pipelines/pipelines-deploy-alternatives.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Pipelines Deploy Alternatives"
+description = "Alternative approaches to deploy lightweight Kubeflow Pipelines to a cluster"
+weight = 15
++++
+
+Despite [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
+whole with many components including pipelines, you also have a choice to deploy
+Kubeflow Pipelines' lite deployment. Pipelines has a kustomize manifests for
```

Comment:
Are we planning to use the word "lite" as a brand for this type of deployment? That's a product management / marketing decision. I recommend that we remove all mention of "lite" and use "lightweight" instead every time.

Reply:
The conclusion is "Deploy Kubeflow Pipelines standalone"/"standalone deployment". I will update the documentation.